pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart

```diff
@@ -413,14 +415,18 @@ class NativeAssetsBuildRunner {
         continue;
       }
       // TODO(https://github.com/dart-lang/native/issues/1321): Should dry runs be cached?
-      var (buildOutput, packageSuccess) = await _runHookForPackage(
-        hook,
-        config,
-        packageConfigUri,
-        workingDirectory,
-        includeParentEnvironment,
-        null,
-        hookKernelFile,
+      var (buildOutput, packageSuccess) =
```

Comment:
nit: `final`

Reply:
Not possible, overwritten below (otherwise it would have been final to begin with, lints should be enabled.)